source/MaterialXContrib/MaterialXMaya/README.md

```diff
@@ -45,44 +49,48 @@ Modifying any one of these attributes via GUI or scripting causes the shading no
 ## Maya Commands
 ### `createMaterialXNode`
 
-Creates one or many MaterialX nodes from the specified MaterialX document. The command accepts the following flags:
+Creates one or many MaterialX nodes from the specified MaterialX document.
+The command accepts the following flags:
 
-```MEL
+```Text
```

Comment:
Not technically a MEL snippet - it's copied from what `help createMaterialXNode` outputs in the log.

Reply:
Does ```MEL even mean anything in Markdown?

Reply:
Yes. This can be the name of any programming language though not all renderers support all languages. Visual Studio Code knows MEL syntax via the MayaCode extension and so it's able to apply MEL syntax highlighting in Markdown preview as well. GitHub sadly doesn't know about MEL but we could fix this by contributing to https://github.com/github/linguist/blob/master/lib/linguist/languages.yml